pkg/virt-launcher/virtwrap/network/selinux_context_executor.go

```diff
@@ -54,23 +54,6 @@ func (ce *SELinuxContextExecution) Execute() error {
 	return nil
 }
 
-func (ce *SELinuxContextExecution) Launch() error {
```

Comment:
General comment - I would change the file name and call it just `selinux`. It has methods for the socket label that are not used by the executor.